src/ui/shared/wallet/signMessage.ts

```diff
@@ -1,11 +1,9 @@
-import { ethers } from 'ethers';
+import { toUtf8Bytes, hexlify } from 'ethers/lib.esm/utils';
```

Comment:
why here is a different type of imports?

Reply:
I started by trying to import directly from sublibs as described here: https://docs.ethers.org/v6/migrating/#migrate-importing

But maybe it's worth to refactor to use the `ethers` global since in v6 everything is in the root package